src/core/Link/LinkList/LinkList.md

```diff
@@ -0,0 +1,44 @@
+### Link List
+
+A styled list for providing a list of links.
+
+Make sure you use a descriptive heading or a label with the list and link it to the list via the `ariaDescribedBy` prop. Make sure you use the right heading level semantically and use the `as` prop for changing the styling if needed.
```

Comment:
This paragraph has quite a lot of information, it could be broken down to a list and/or separate examples. I'd lose the "make sure you..." parts to keep instructions shorter.

For example

- Use a descriptive heading or a label
- Link the label or heading to the list via...
- Use right heading level...
- Use as prop...

`use the 'as' prop for changing styling` -> this could be it's own example

Reply:
Improved the instructions somewhat. I don't yet see the point of adding the as prop use as a separate example since it's now both explained verbally and included in the first and only example without cluttering it too much IMO.